var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -12,26 +12,29 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
     portable applications targeting all major HPC platforms."""
 
     homepage = "https://github.com/kokkos/kokkos"
-    git = "https://github.com/kokkos/kokkos.git"
-    url      = "https://github.com/kokkos/kokkos/archive/3.4.01.tar.gz"
+    git      = "https://github.com/kokkos/kokkos.git"
+    url      = "https://github.com/kokkos/kokkos/archive/3.5.00.tar.gz"
 
     tags = ['e4s']
 
     test_requires_compiler = True
 
-    maintainers = ['DavidPoliakoff', 'jciesko']
+    maintainers = ['janciesko', 'crtrott']
```

Comment:
You might want to pick someone who actually uses Spack.

Reply:
Honestly I put Christian to motivate him to find someone at the next developers meeting, I don't feel it's my decision to make...